yarn-project/aztec-cli/src/index.ts

```diff
@@ -94,7 +94,7 @@ export function getProgram(log: LogFn, debugLogger: DebugLogger): Command {
   program
     .command('generate-private-key')
     .description('Generates a 32-byte private key.')
-    .option('-m, --mnemonic', 'A mnemonic string that can be used for the private key generation.')
+    .requiredOption('-m, --mnemonic', 'A mnemonic string that can be used for the private key generation.')
```

Comment:
It seems this isn't required? There's an `else` statement which would succeed if no option is given.

Reply:
You are right. Addressed in [bc22c85](https://github.com/AztecProtocol/aztec-packages/pull/1584/commits/bc22c85d72da1551c672b0d826fa9c7bb2287bf9)